### PR TITLE
Feature/us13 add conjugation

### PR DESCRIPTION
**Pull request by ufelgen**
Following [this US](https://github.com/ufelgen/capstone-project/issues/23) I added an option to save verb conjugations. For demonstration purposes, I hard coded a full conjugation for one verb (peči), which is marked by a little grey "C" icon. A function carries out the conjugation otherwise, but I'm certain it will be incorrect for some irregular verbs. Therefore, the next US will be to edit the conjugation pages. 

Happy new year everyone!

### CODE REVIEW

File: components/ConjugationButtons/ConjugationButtons.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export default function ConjugationButtons({ tense, onChangeTense }) {
+  return (
+    <div>
+      <StyledTenseButton
+        type="button"
+        className={tense === "past" ? "current" : ""}
```
Comment (AntjeWalter) [inline components/ConjugationButtons/ConjugationButtons.js:8]: Wow, I never thought about using the ternary operator for a className. Nice!

File: components/ConjugationPage/ConjugationPage.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              ))}
+          </tbody>
+        </StyledTable>
+      </StyledSection>
+    </>
+  );
+}
+
+const StyledTable = styled.table`
+  padding: 0.625rem;
```
Comment (AntjeWalter) [inline components/ConjugationPage/ConjugationPage.js:50]: okay, THAT's a pretty specific measurement :D

File: components/PopupMenu/PopupMenu.js [hunk 3]
```diff
@@ -25,6 +25,11 @@ export default function PopupMenu({
               {prop.query1?.declension ? "declension" : "+ declension"}
             </StyledMenuButton>
           </Link>
+          <Link href={`/conjugation/${id}`}>
+            <StyledMenuButton>
+              {prop.query1?.conjugation ? "conjugation" : "+ conjugation"}
+            </StyledMenuButton>
```
Comment (AntjeWalter) [inline components/PopupMenu/PopupMenu.js:31]: I love this, its such a nice feature in such a small code :)

**Review by AntjeWalter — APPROVED** [review]
Not much to add, looks pretty impressive to me! :)

File: components/ConjugationPage/ConjugationPage.js (hunk 2 above)
Comment (ufelgen) [reply components/ConjugationPage/ConjugationPage.js:50]: I had pixels before - just calculated from there ;)

**Pull request merged**